src/Buildalyzer/AnalyzerResult.cs

```diff
@@ -370,8 +371,8 @@ private static IEnumerable<string> EnumerateCommandLinePartsFsc(string commandLi
 
         private class ProjectItemItemSpecEqualityComparer : IEqualityComparer<IProjectItem>
         {
-            public bool Equals(IProjectItem x, IProjectItem y) => x.ItemSpec.Equals(y.ItemSpec);
-            public int GetHashCode(IProjectItem obj) => obj.ItemSpec.GetHashCode();
+            public bool Equals(IProjectItem x, IProjectItem y) => x.ItemSpec.Equals(y.ItemSpec, StringComparison.CurrentCultureIgnoreCase);
+            public int GetHashCode(IProjectItem obj) => obj.ItemSpec.ToLower().GetHashCode();

```

Comment:
```suggestion
            public int GetHashCode(IProjectItem obj) => obj.ItemSpec.ToLowerInvariant().GetHashCode();

```

Reply:
Yep, I agree with @slang25 we should be using `OrdinalIgnoreCase` everywhere `CurrentCultureIgnoreCase` was added. Nice catch. Also agree with `ToLowerInvariant`.